rose/web/game.js

```diff
@@ -327,13 +327,19 @@ var ROSE = (function() {
     }
 
     Cars.prototype.draw = function(ctx) {
+        ctx.fillStyle = "rgb(0, 0, 0)";
+        ctx.textBaseline = "center";
```

Comment:
We should use "top" here - center is not valid value.
See https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/textBaseline